moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectWebsite.js

```diff
@@ -113,15 +114,14 @@ const ProjectSummaryProjectWebsite = ({
         )}
         {!editMode && (
           <ProjectSummaryLabel
-            className={classes.fieldLabelLink} // Override
             text={
               (website && website.length > 0 && (
                 <Link href={website} target={"_blank"}>
                   {website}
                   <OpenInNew className={classes.linkIcon} />
                 </Link>
               )) ||
-              "None"
+              ""
```

Comment:
I find that I get displayed a `-` when I set the website to the empty string. Are we storing a website URL as the empty string when we mean `null`? Maybe I'm confused about where the `-` is coming from for this field.

Reply:
I was curious too, but in the External Link component, if theres no text it renders as `-`